lib/shopify_api/rest/request.ex

```diff
@@ -205,28 +205,21 @@ defmodule ShopifyAPI.REST.Request do
     ]
   end
 
-  @spec add_params_to_url(binary, list | map) :: binary
-  defp add_params_to_url(url, params) when is_map(params) do
-    list_params = Enum.map(params, fn {key, value} -> {String.to_existing_atom(key), value} end)
-    add_params_to_url(url, list_params)
-  end
-
+  @spec add_params_to_url(url :: binary, params :: list | map) :: binary
   defp add_params_to_url(url, params) do
     url
     |> URI.parse()
     |> merge_uri_params(params)
     |> to_string()
   end
 
-  @spec merge_uri_params(URI.t(), list) :: URI.t()
-  defp merge_uri_params(uri, []), do: uri
-
+  @spec merge_uri_params(URI.t(), params :: list | map) :: URI.t()
   defp merge_uri_params(%URI{query: nil} = uri, params) when is_list(params) or is_map(params) do
     Map.put(uri, :query, URI.encode_query(params))
   end
 
   defp merge_uri_params(%URI{} = uri, params) when is_list(params) or is_map(params) do
-    Map.update!(uri, :query, fn q ->
+    Map.update!(uri, :query, fn query ->
```

Comment:
🔥 renamed the param here but not use in the func

Reply:
fuuuuu

Reply:
Quick on the draw 🤠